opensfm/mesh.py

```diff
@@ -20,12 +20,20 @@ def triangle_mesh(shot_id, r, graph, data):
         return triangle_mesh_perspective(shot_id, r, graph)
     elif shot.camera.projection_type == 'fisheye':
         return triangle_mesh_fisheye(shot_id, r, graph)
+    elif shot.camera.projection_type == 'dual':
+        return triangle_mesh_dual(shot_id, r, graph)
     elif shot.camera.projection_type in ['equirectangular', 'spherical']:
         return triangle_mesh_equirectangular(shot_id, r, graph)
     else:
         raise NotImplementedError
 
 
+def triangle_mesh_dual(shot_id, r, graph):
+    shot = r.shots[shot_id]
+    cam = shot.camera
```

Comment:
We do not seem to need this. Maybe we can skip that function and just call the fisheye version directly